packages/eslint-plugin-lit-a11y/lib/rules/aria-role.js

```diff
@@ -50,21 +51,23 @@ const AriaRoleRule = {
                   return;
                 }
 
-                if (rawValue.startsWith('{{')) {
+                const val = getExpressionValue(analyzer, rawValue);
+
+                if (!val && rawValue.startsWith('{{')) {
```

Comment:
maybe a helper `isInterpolation`?

Reply:
yeah we can do, id prefer the name `isPlaceholder` though